src/status_im/native_module/impl/module.cljs

```diff
@@ -63,7 +63,9 @@
                                  "JavaScriptCore"
                                  "OttoVM")
                                " jail initialized")]
-          (.initJail status init-js' #(do (re-frame/dispatch [:initialize-app])
+          (.initJail status init-js' #(go (re-frame/dispatch [:initialize-db])
+                                          (<! (timeout 500))
```

Comment:
Maybe not the best to rely on waiting, as we can only assume 500 ms is enough, if the two events are so coupled probably having a single handler or make one trigger the other is a better way, what do you think?

Reply:
Agreed